docs/howtos/vpn.md

```diff
@@ -61,92 +58,77 @@ Using the nsenter command:
 $ sudo nsenter --net --target $(pidof openvpn) -- ip link show vpn0
 ```
 
-## Share environment network
+## Share network between applications
 
-When joining network directly, the process may loose connectivity when
+When joining the network manually, the session may loose connectivity when
 the openvpn container dies. And when openvpn get restarted, the foreign processes
 need to rejoin the new namespace.
 
 Podenv can provide a persistent network namespace by using an *infra*
 container to keep the namespace alive. This can be activated by using the
-`--net` command line with an arbritary name.
+`--namespace` command line.
+
 Restart the **openvpn** container using:
 
 ```console
-$ podenv --home ~/.config/openvpn --net ovpn openvpn ovpn.conf
+$ podenv --home ~/.config/openvpn --namespace ovpn openvpn ovpn.conf
 ```
 
-Then other environment can join the namespace, for example:
+Then other application can join the namespace, for example:
 
 ```console
-$ podenv --net ovpn firefox
+$ podenv --namespace ovpn firefox
 ```
 
-With that setup, the openvpn container can be restarted when needed, and
-the other environment will recover the connectivity transparently.
+Using podenv namespace, the openvpn container can be restarted when needed, and
+the other applications will recover the connectivity transparently.
```

Comment:
Ah really ? It is not the case today with my current podenv legacy setup.

Reply:
That should work as described because of the infra container.